app/backend/src/couchers/models.py

```diff
@@ -131,6 +132,10 @@ class User(Base):
     def is_jailed(self):
         return self.accepted_tos < 1 or self.is_missing_location
 
+    @hybrid_property
+    def is_hidden(self):
+        return self.is_banned and self.is_jailed and self.is_deleted
```

Comment:
These should be ORs, but you need to use the `|` (bitwise) OR so that sqlalchemy can do its magic.

Reply:
oops. duh

Reply:
I just re-read what you wrote and realized that I completely misunderstood. Fixing now.